src/bin/lpython.cpp

```diff
@@ -1614,7 +1614,7 @@ int main(int argc, char *argv[])
         app.require_subcommand(0, 1);
         CLI11_PARSE(app, argc, argv);
 
-        lcompilers_unique_ID = LCompilers::get_unique_ID();
+        lcompilers_unique_ID = "__" + LCompilers::get_unique_ID();
```

Comment:
I would keep the `__` out of the unique ID and leave it to the use case to insert this properly.

Reply:
Okay! done, thanks!